docs/_includes/jlbp-header.html

```diff
@@ -0,0 +1,61 @@
+<!-- prettier-ignore -->
+<header class="glue-header glue-header--single">
+  <div class="glue-header__bar glue-header__bar--desktop glue-header__drawer">
+    <div class="glue-header__tier">
+      <!-- LOCK up-->
+      <div class="glue-header__container">
+        <div class="glue-header__lock-up">
+          <div class="glue-header__logo">
+            <a class="glue-header__logo-link" href="{{ '/' | relative_url }}" title="Google">
+              <div class="glue-header__logo-container">
+                <div class="cloud-logo">
+                  <img src="/assets/images/Cloud_Logo_Nav.svg" alt="Google Cloud">
+                </div>
+              </div>
+              <p class="glue-header__logo--product">JLBPs</p>
+            </a>
+          </div>
+          <a href="#page-content" class="glue-header__skip-content">
```

Comment:
"Error

Interactive elements must be labeled. Learn more"

Reply:
This is taken straight from the Glue sample: https://glue-docs.appspot.com/docs/components/raw/header#HTML

Reply:
I guess I don't trust Glue as much as you do. When Firefox scans aip.dev it finds a number of problems I know are real issues, so I'm inclined to give it the benefit of the doubt on the ones I don't immediately understand. If you like, we can dig into these further, though in general I'm more comfortable building up from small pieces of code I understand than trying to cut out the large pieces I don't.

Reply:
If there are problems with Glue, I think we should get them fixed at the source instead of 1) trying hack fixes on top of Glue or 2) forging our own path. I would argue that fixing problems at the source should not be in the scope of this PR and shouldn't block it, given the value-add nature of the PR. Are you going to set a bar of perfection in order to approve the PR? We are not a front-end team; we are a team trying to help people solve diamond dependency conflicts.

I would guess that the accessibility issues on aip.dev that have been fixed on jlbp.dev are likely local choices and not issues with Glue.